tests/boot/boot_from_pxe.pm

```diff
@@ -109,10 +119,25 @@ sub run {
     save_screenshot;
 
     if ((check_var('BACKEND', 'ipmi') && !get_var('AUTOYAST')) || get_var('SES5_DEPLOY')) {
-        my $ssh_vnc_wait_time = 600;
-        assert_screen((check_var('VIDEOMODE', 'text') ? 'sshd' : 'vnc') . '-server-started', $ssh_vnc_wait_time);
-        select_console 'installation';
+        my $ssh_vnc_wait_time = get_var('SES5_DEPLOY') ? 300 : 210;
```

Comment:
looks like you are reverting a change from a previous PR which came in in the mean time, please revert

Reply:
> looks like you are reverting a change from a previous PR which came in in the mean time, please revert

Which one ?

Reply:
You are changing back the timeout from 600s to being 300/210s depending on the variable SES5_DEPLOY. The last commit that bumped the overall timeout from 300s to 600s was e6df299b9, the former getting rid of the variable check was 47c045090 from 27th of September.